packages/lab/src/tabs-next/TabNext.tsx

```diff
@@ -0,0 +1,159 @@
+// TODO close button needs to be a button. Hence tab needs to include 2 buttons
+import { Button, ButtonProps, makePrefixer, useForkRef } from "@salt-ds/core";
+import { CloseIcon, CloseSmallIcon } from "@salt-ds/icons";
+import { clsx } from "clsx";
+import {
+  forwardRef,
+  ForwardedRef,
+  KeyboardEvent,
+  MouseEvent,
+  ReactElement,
+  useCallback,
+  useRef,
+  useState,
+} from "react";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { TabProps } from "./TabsNextTypes";
+
+import tabCss from "./TabNext.css";
+
+const withBaseName = makePrefixer("saltTabNext");
+
+//TODO not ideal - duplicating the Icon then hiding one in css based on density - is there a nicer way ?
```

Comment:
would a density provider make sense here?

Reply:
How does density provider help here? Not sure I understand 🤔

Reply:
show or hide based on density?

Reply:
Ok, so do a conditional render in javascript? I don't see how that is a better solution than a css conditional display.